### ISSUE TITLE
Add a directory structure graphic in README.md

### ISSUE THREAD

**Issue by hemangsk** [hemangsk/Gravity#16]
You have to create a directory structure  graphic in README.md using markdown and write in one line what each folder contains. 

**Update**
Directory structure created. Add one line per folder explaining the contents of the folder.

Example of a directory structure : (from https://raw.githubusercontent.com/mmistakes/made-mistakes-jekyll/master/README.md)

``` bash
├── gulp                      # => gulp tasks
├── src                       # => source Jekyll files and assets
|  ├── _includes
|  ├── _layouts
|  ├── _plugins
|  ├── ...
|  ├── _posts
|  ├── assets
|  |  ├── icons
|  |  ├── images
|  |  |   └── feature
|  |  ├── javascript
|  |  |   ├── plugins
|  |  |   ├── vendor
|  |  |   └── main.js
|  |  ├── stylesheets
|  |  |   ├── vendor
|  |  |   ├── ...
|  |  |   └── style.scss
├── .editorconfig
├── .gitignore
├── _config.dev.yml
├── _config.yml
├── Gemfile
├── gulpfile.js
├── package.json
├── rsync-credentials.json
├── ...
```

That's it! 
Got Queries? Need help getting started ? Just comment below.

**Comment by imadhsissou** [comment]
I can take this ! give me few moments ;)

**Comment by hemangsk** [comment]
Solved in #20 , #18  by @7BISSO and @jcardoz  🎉
